.github/workflows/build.yml

```diff
@@ -181,6 +181,21 @@ jobs:
     outputs:
       dependency_cache_key: ${{ steps.compute_lockfile_hash.outputs.hash }}
 
+  job_check_branches:
+    name: Check PR branches
+    needs: job_get_metadata
+    runs-on: ubuntu-20.04
+    if: github.event_name == 'pull_request'
+    permissions:
+      pull-requests: write
+    steps:
+      - name: PR is opened against master
+        uses: mshick/add-pr-comment@v2
+        if: ${{ github.base_ref == 'master' && github.head_ref != 'develop' }}
+        with:
+          message: |
+            ⚠️ This PR is opened against **master**. You probably want to open it against **develop**.
```

Comment:
l: Should we ignore this for `prepare-release` branches?

Reply:
ah, good point, the head_ref is prepare-release then, will adjust this!